_template/bin/nuget

```diff
@@ -1,2 +1,6 @@
-result = system(File.dirname(__FILE__) + "/nuget.exe " + ARGV.join(' '))
-exit 1 unless result
\ No newline at end of file
+host_os = RbConfig::CONFIG['host_os']
+windows = /mswin|msys|mingw|cygwin|bccwin|wince|emc/.match(host_os)
```

Comment:
That's a large set of choices - are they all still relevant? I know bundler really only looks at the first three. Interesting to select on cygwin as well.